### PR TITLE
Add panel color configuration

### PR DESCRIPTION
**Pull request by nagata-yoshiteru**
## Description

Add panel color configuration so that users can customize it by their preference.

### Changes
- Add extension configuration
- Change panel color based on the config

### CODE REVIEW

**Comment by nagata-yoshiteru** [comment]
Hi, @BenWeisz!
I've added the panel color customization feature. Let me know what you think!
Thanks.

File: src/webview.ts [hunk 1]
```diff
@@ -24,7 +21,9 @@ const generateHTMLCanvas = (
   data: string,
   width: number,
   height: number,
-  imgType: string
+  imgType: string,
+  bgColor: string,
+  btnColor: string
 ): string => {
```
Comment (weiszben) [inline src/webview.ts:27]: I think the code would be a bit cleaner if we move the css object into the function and then insert the colors into the class strings

File: src/webview.ts [hunk 2]
```diff
@@ -34,16 +33,16 @@ const generateHTMLCanvas = (
...
+        <div style="${styles.info} background-color: ${bgColor};">
           <p>Type: ${imgType}</p>
           <p>Width: ${width}px</p>
           <p>Height: ${height}px</p>
           <p id="scale-display">Zoom: 100%</p>
           <div style="margin-bottom: 5px">
-            <div onclick="scale = scale * 2; showImg(scale);" style="${styles.sizingButton}">+</div>
-            <div onclick="scale = scale / 2; showImg(scale);" style="${styles.sizingButton}">-</div>
+            <div onclick="scale = scale * 2; showImg(scale);" style="${styles.sizingButton} background-color: ${btnColor};">+</div>
+            <div onclick="scale = scale / 2; showImg(scale);" style="${styles.sizingButton} background-color: ${btnColor};">-</div>
```
Comment (weiszben) [inline src/webview.ts:43]: By doing the above, we could remove the duplicate `background-color` tag here

**Review by weiszben — CHANGES_REQUESTED** [review]

File: src/webview.ts (hunk 1 above)
Comment (nagata-yoshiteru) [reply src/webview.ts:27]: Ok, I'll try to do that!

**Review by nagata-yoshiteru — COMMENTED** [review]
@BenWeisz 
I refactored the code based on your suggestion!
Thanks!

**Review by BenWeisz — APPROVED** [review]

**Pull request merged**